ansible/tasks/postgres-extensions/21-pg_jsonschema.yml

```
@@ -0,0 +1,51 @@
+# pg_jsonschema
+- name: pg_jsonschema - download & install dependencies
+  apt:
+    pkg:
+      - make
+      - gcc
+      - pkg-config
+      - clang
+      - postgresql-server-dev-{{ postgresql_major }}
+      - libssl-dev
+    update_cache: yes
+    install_recommends: no
+
+- name: check if cargo is installed
+  shell: command -v cargo
+  register: cargo_exists
+  ignore_errors: yes
+
+- name: download rust/cargo installer
+  when: cargo_exists is failed
+  get_url:
+    url: https://sh.rustup.rs
+    dest: /tmp/sh.rustup.rs
+    mode: '0755'
+    force: 'yes'
+  tags:
+    - rust
+
+- name: install rust/cargo
+  when: cargo_exists is failed
+  shell: /tmp/sh.rustup.rs -y
+  tags:
+    - rust
+
+- name: install cargo-pgx
+  shell:
+    cmd: "cargo install --version '={{ pgx_version }}' cargo-pgx && cargo pgx init --pg{{ postgres_major }} pg_config"
```

```suggestion
    cmd: "~/.cargo/bin/cargo install --version '={{ pgx_version }}' cargo-pgx && ~/.cargo/bin/cargo pgx init --pg{{ postgres_major }} pg_config"
```

At this point, `cargo` is not yet included in the PATH environment. We'll just run it from the default directory `~/.cargo/bin/cargo`